### ISSUE TITLE
Create option to test branch on dedicated url's

### ISSUE THREAD

**Issue by StudioSpindle** [dorel/dorel.github.io#66]
TL;DR: create GIT branch -> finish work, distribute to automatically fired server instance -> test -> option to distribute into main branch -> happy

When working on an issue, it can be the case that you'd people to be able to test a specific branch on the site without inflicting changes to other parts. 

Distributing these changes to a central development environment can conflict with BAT for other sections in the mean time. 

Would be ideal if you can create a new git branch, and then distribute the branch to a dedicated URL (example: gh-xx.dorel-test.io). 

This way you make the changes available to everyone to test.
